packages/dds/map/src/mapKernel.ts

```diff
@@ -41,7 +41,7 @@ interface IMapMessageHandler {
      */
     submit(op: IMapOperation, localOpMetadata: unknown): void;
 
-    getStashedOpLocalMetadata(op: IMapOperation): unknown;
+    processStashedOpLocalMetadata(op: IMapOperation): unknown;
```

Comment:
maybe just match the name on shared object, `applyStashedOp`

Reply:
it can be nice when searching for concepts for them to be called similarly, it also conveys the intention of the method in that it is the same as other methods with the same name

Reply:
Also, processStashedOpLocalMetadata sounds like it's processing the local metadata rather than also getting it.